bluesky_widgets/models/run_engine_client.py

```diff
@@ -58,6 +61,13 @@ def __init__(self, zmq_server_address=None, user_name="GUI Client", user_group="
         #   using positions is consistent. Empty list - no items are selected
         self._selected_history_item_pos = []
 
+        # Current directory (used for opening/saving files).
+        self.current_dir = None
+        # Set the variable to the name of the Queue Server Custom Module (if available)
+        self.qserver_custom_module_name = None
```

Comment:
Should this be a list of modules?

Reply:
No, just the name of the module or None if there is no such module. I am using environment variable to pass the module name to HTTP server.